components/ActionSheet/src/MDCActionSheetController.h

```diff
@@ -203,6 +203,22 @@ __attribute__((objc_subclassing_restricted)) @interface MDCActionSheetController
  */
 @property(nonatomic) UIImageRenderingMode imageRenderingMode;
 
+/**
+ Determines if a divider should be shown between the header and actions. To customize the divider
+ color see @c headerDividerColor.
+
+ Defaults to NO.
+ */
+@property(nonatomic, assign) BOOL showsHeaderDivider;
+
+/**
+ The color of the divider between the header and actions. This API should be used with
+ @c showsHeaderDivider to make the divider visable.
+
+ Defaults to a ligh grey color.
```

Comment:
Either this comment or the code below is incorrect. The color is not a light grey color, but a partially-transparent black color.  "Light grey" generally means an opaque white color with a white value greater than 0.5.

Perhaps `[UIColor colorWithWhite:0.88 alpha:1]` or something similar?  As an added benefit, opaque colors often render faster because no blending has to take place.

Reply:
Updated the docs.